### ISSUE TITLE
HD 143006 Tutorial Part 2: no attribute 'TSV'

### ISSUE THREAD

**Issue by astromark** [MPoL-dev/MPoL#94]
I've just been trying out the HD 143006 tutorial. In part 2, in the training loop section I run into the following error when calling train.

```
In [42]: train(model, dataset, optimizer, config, writer=writer)                
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-42-7caa7aa1aa20> in <module>
----> 1 train(model, dataset, optimizer, config, writer=writer)

<ipython-input-36-e9bbcde80bd3> in train(model, dataset, optimizer, config, writer, logevery)
     15             + config["lambda_TV"] * losses.TV_image(sky_cube)
     16             + config["entropy"] * losses.entropy(sky_cube, config["prior_intensity"])
---> 17             + config["TSV"] * losses.TSV(sky_cube)
     18         )
     19 

AttributeError: module 'mpol.losses' has no attribute 'TSV'
```

I guess the version of mpol.losses used to write the tutorial is different to the one currently available via pip. I did get the rest of the tutorial to work by removing the TSV line from train, although the final image is much worse than the one in the tutorial and even worse than the dirty image, so I guess the TSV is important.

**Comment by iancze** [comment]
Yes, I think that's what's going on. The tutorials and documentation are built from the master branch. We'll be releasing a new pip version every time we close out a feature branch, but if you want to try out the feature you can clone the github repo and install from source. It should work fine (since it's continuously integrated, and currently passing), but if not, please raise an issue.

**Comment by iancze** [comment]
This should be fixed by the new v0.1.2 release on pip.
